DEVELOPMENT.md

```diff
@@ -1,11 +1,14 @@
 Table of Contents
 ----------------
 
+- [Setup](#setup)
+    - [Get source code](#get-source-code)
```

Comment:
I half-reverted so that we move "Build" part only. It is bit awkward to have "get source" and "build" chapters in different documents, but thats what we get now unless we want to repeat.